README.md

```diff
@@ -19,8 +19,9 @@ You can ask questions about Dafny on [Stack Overflow](https://stackoverflow.com/
 
 # Try Dafny
 
-The easiest way to get started with Dafny is to use [rise4fun](http://rise4fun.com/dafny), where you can write and verify Dafny programs without having install anything. On [rise4fun](http://rise4fun.com/dafny), you will also find the online Dafny tutorial.
-It is also easy to [install Dafny on your own machine](https://github.com/dafny-lang/dafny/wiki/INSTALL) in VS Code, which gives you a much better user experience than in the web browser.
+The easiest way to try out Dafny is to [download](https://github.com/dafny-lang/dafny/releases) it
+to run it on your machine as you follow along with the [Dafny tutorial](https://dafny-lang.github.io/dafny/OnlineTutorial/guide).
+It is also easy to [install Dafny on your own machine in VS Code](https://github.com/dafny-lang/dafny/wiki/INSTALL).
```

Comment:
I suggest we first propose VS Code. We can then add, "If you prefer to run the Dafny tool from the command line, download it from here...".

I'm not sure how to refer to them from here, but viewing the four 20-minute videos on the documentation page and following along in VS Code is also a good way to learn basic things.